caffe2/python/convnet_benchmarks.py

```diff
@@ -654,7 +692,10 @@ def AddParameterUpdate(model):
 
 
 def Benchmark(model_gen, arg):
-    model, input_size = model_gen(arg.order, arg.cudnn_ws, arg.model_path)
+    if arg.model == 'Resnet50':
```

Comment:
You won't need this once you change the method signature to Resnet50(order, cudnn_ws, model_path="")